src/v/cluster/node/local_monitor.h

```diff
@@ -24,12 +25,13 @@ namespace cluster::node {
 
 class local_monitor {
 public:
-    local_monitor() = default;
-    local_monitor(local_monitor&) = delete;
+    local_monitor(
+      config::binding<size_t> min_bytes, config::binding<unsigned> min_percent);
+    local_monitor(local_monitor&) = default;
```

Comment:
needing a copy constructor for stuff like this is pretty rare. why is it needed here?

also, shouldn't the reference be const if this is a copy constructor?